inst/private/check_and_convert.m

```diff
@@ -56,10 +58,14 @@
       % make sure values are converted to sym
       s = structfun (@(t) check_and_convert (t){:}, s, 'UniformOutput', false);
       obj{i} = s;
-    elseif (py.isinstance(x, pyeval('int')))
+    elseif (py.isinstance(x, integer_types))
       if (py.isinstance(x, pyeval('bool')))
         error ('unexpected python bool')
       end
+      if (abs (r) > intmax ('int64'))
```

Comment:
`r` is undefined here, maybe `if (abs (double (x)) > intmax ('int64'))`?

This is the kind of range test I feared might be too costly.